cllc-public-app/ClientApp/src/app/models/establishment.model.ts

```diff
@@ -1,5 +1,6 @@
 export class Establishment {
   id: string;
-  phone: string;
-  email: string;
+  phone?: string;
+  email?: string;
```

Comment:
Can you comment on the reason for this change?

Reply:
I made them nullable so that I can send update requests with just a few of the fields. If you think we should leave them non-nullable I can of course send the data that is currently attached to the establishment.

Reply:
Can you remove the question mark from the two strings then, to be consistent with other code.  I believe all strings are nullable in TypeScript / JavaScript.   For the boolean that is optional the question mark should remain.

Reply:
updated